sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/SessionRecords/DataFeedLiveTests/UpdateAzureApplicationInsightsDataFeedWithEveryMemberAndGetInstance.json

```diff
@@ -8,11 +8,8 @@
         "Content-Length": "284",
         "Content-Type": "application/json",
         "Ocp-Apim-Subscription-Key": "Sanitized",
-        "traceparent": "00-dd2b99d0f5f21f4f89d0c1a041f8602f-f94909c9513c0140-00",
```

Comment:
Tests needed to be rerecorded because service now returns the updated data feed. Originally the response body was empty.